package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@studiometa/create-wordpress-project",
-  "version": "1.0.0-beta8",
+  "version": "1.0.0-beta9",
```

Comment:
**issue:** Change `beta9` to `beta.9`

The beta version format for NPM packages is `vX.X.X-beta.X`.